packages/evm/src/interpreter.ts

```diff
@@ -11,7 +12,11 @@ import {
 } from '@ethereumjs/util'
 import debugDefault from 'debug'
 
-import { EOF } from './eof.js'
+//import { EOF } from './eof/eof.js'
```

Comment:
Same, can we remove?

Reply:
Oh wait this might actually be a leftover of old code (will check)